db/migrations/2023091117500001-add-governor-to-allow_list-column.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /** todo: this can be made into one single sql query */
+
+    const curatorsWithNetworkDbWithAllowListEmpty = await queryInterface.sequelize
+      .query(`select "creatorAddress", id from networks where array_length(allow_list, 1) = 0`);
+
+    for (const row of curatorsWithNetworkDbWithAllowListEmpty)
+      await queryInterface.sequelize.query(`update networks set allow_list = array (row.creatorAddress) where id = row.id`);
```

Comment:
I think the values of `row` aren't being passed to the query. Maybe it's necessary to bind the values, or use `${}` to mount the query string.

Reply:
@vhcsilva i was thinking and this migration doesn't make sense because as soon as it enters it makes it that no one else can open a bounty. Ill just remove this and the users will have to add it by hand if wanted.